pipeline_template.yaml

```diff
@@ -0,0 +1,37 @@
+---
+version:
+
+date:
+
+pipelines:
+  - models:
+    model_name:
+    pipeline_name:
+    created_by:
+    creation_date:
+    version:
+    description:
+    job_name:
+    dataset:
+    metrics:
+    identifiable:
+    default:
+
+models:
+  - tasks:
+    api_call_function:
+    endpoint:
+    api_key:
+    inference_function:
+    content_type:
```

Comment:
what does this mean exactly?

Reply:
`content_type` is a string representation of what is sent to the model, in our case its always an `application/json`